api.go

```diff
@@ -528,109 +529,109 @@ func (sc *Client) UpdateSite(corpName, siteName string, body UpdateSiteBody) (Si
 	return site, nil
 }
 
-// CustomAlert contains the data for a custom alert
-type CustomAlert struct {
-	ID        string
-	SiteID    string
-	TagName   string
-	Interval  int
-	Threshold int
-	Enabled   bool
-	Action    string
-	Created   time.Time
+// CreateCustomAlertBody is the body for creating a custom alert.
+type CreateCustomAlertBody struct {
+	TagName              string `json:"tagName,omitempty"`    //The name of the tag whose occurrences the alert is watching. Must match an existing tag
+	LongName             string `json:"longName,omitempty"`   //A human readable description of the alert. Must be between 3 and 25 characters.
+	Interval             int    `json:"interval"`             //The number of minutes of past traffic to examine. Must be 1, 10 or 60.
+	Threshold            int    `json:"threshold"`            //The number of occurrences of the tag in the interval needed to trigger the alert.
+	BlockDurationSeconds int    `json:"blockDurationSeconds"` //The number of seconds this alert is active.
+	Enabled              bool   `json:"enabled"`              //A flag to toggle this alert.
+	Action               string `json:"action,omitempty"`     //A flag that describes what happens when the alert is triggered. 'info' creates an incident in the dashboard. 'flagged' creates an incident and blocks traffic for 24 hours.
 }
 
-// customAlertsResponse is the response for the alerts endpoint
-type customAlertsResponse struct {
-	Data []CustomAlert
+//ResponseCustomAlertBody contains the data for a custom alert
+type ResponseCustomAlertBody struct {
```

Comment:
I like your naming system better, but this will break compile for any existing clients. Is it worth that?
And if so, can you include a comment for the new structures to say what the structure's previous name was so it's easier for anyone trying to fix their own code.